File: dayOne/java/levelThree/FlattenArray.java
public class FlattenArray {

    public static int getElementsCount(int[][] array) {
        int total = 0;

        for (int[] element : array) {
            total += element.length;
        }

        return total;
    }

    public static int[] flatten(int[][] array) {

        int elementCount = getElementsCount(array);

        int[] flattenedArray = new int[elementCount];
        int flattenedArrayIndex = 0;

        for (int index = 0; index < array.length; index++)
            for (int innerIndex = 0; innerIndex < array[index].length; innerIndex++)
                flattenedArray[flattenedArrayIndex++] = array[index][innerIndex];

        return flattenedArray;
    }
}

File: dayOne/python/levelThree/flatten_array.py
def get_total_element_count(array):
    total = 0

    for element in array:
        total += len(element)

    return total

def flatten(array):
    flattened_array = []

    for listItems in array:

        for items in listItems:
            flattened_array.append(items)

    return flattened_array